client/src/app/pages/login/login.page.html

```diff
@@ -0,0 +1,14 @@
+<ion-header>
+  <app-header>
+    <ion-title>Login</ion-title>
```

Comment:
I think we could even re-use the submit-button-label-text for this, as this page will not contain a lot of other translatable elements... ;)